### ISSUE TITLE
The loss is so large and iou of each class is 0 except background.

### ISSUE THREAD

**Issue by yyangmei** [meetshah1995/pytorch-semseg#156]
I train segnet using SBD and pascal voc 2012, but my loss is so large such as 578002. And when i eval my model on val, i get iou of each class is 0. What's wrong is it? Thank you so much.

![image](https://user-images.githubusercontent.com/42143920/48323959-b35d7080-e669-11e8-8130-936c501044e4.png)

**Comment by yunclouding** [comment]
i have the same question,  do you have any solution

**Comment by pnzsl** [comment]
> set **size_average=true** in the config file

this is my config.would you mind telling me how to resolve the problem in above?
![image](https://user-images.githubusercontent.com/44644998/63416571-39e18280-c432-11e9-94e9-a150e6bb74a3.png)
loss is large and acc is bad

**Comment by opee007** [comment]
i have the same question, do you have any solution?
